SI/Sem.04/Pract.04/Task14.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+using namespace std;
+int sumOfDigits(long long num)
+{
+	int digits = 0;
+	int numCopy = num;
+	int sum = 0;
+	while (numCopy != 0)
+	{
+		numCopy /= 10;
+		digits++;
```

Comment:
You can directly sum the digits while you divide the number, in that way you will skip the next for loop